frontend/src/components/browser/inspectorBrowser.js

```diff
@@ -335,6 +338,16 @@ class InspectorBrowser extends Component {
     );
   }
 
+  /* */
+  checkFilteredClip(clip) {
+    /*
+    Checks if clip is filtered
+    */
+    if (this.props.filter.clips.includes(clip.id)) {
+      return true;
+    }
```

Comment:
Don't forget to return false since this can cause unexpected behavior.
I recommend changing the function to:
`return this.props.filter.clips.includes(clip.id)`